dask/dataframe/tseries/resample.py

```diff
@@ -28,7 +28,13 @@ def _resample_series(
     how_args,
     how_kwargs,
 ):
+    base_date = start - pd.tseries.frequencies.to_offset(rule)
+    series.loc[base_date, :] = None
```

Comment:
Gonna think on this a bit :)

I dislike that this coerces int dtypes to float, boolean to object, etc.

Does this `base_date` always end up by itself outside the first window? So that it won't affect the result of any computation?

Reply:
Right it makes one extra bin at the beginning. Honestly this is probably something that should be handled on the pandas side. All the answers on the SO post were hacky.

Reply:
Maybe the value that should be appended at the beginning could come from meta?

Reply:
Yeah, that's likely better. As long as the value we're inserting doesn't affect the values going in (by changing the dtype) and it doesn't end up in any of the "real" windows, then this may be OK.

And agreed that it should be handled on the pandas side :)